packages/mml-document/package.json

```diff
@@ -9,10 +9,11 @@
     "lint:fix": "eslint \"./src/**/*.{js,jsx,ts,tsx}\" --fix"
   },
   "dependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "@mml-io/mml-react-types": "^0.0.0-experimental-d10f2a9-20230814",
```

Comment:
You should be able to use `0.5.0` now: https://github.com/mml-io/mml/releases/tag/v0.5.0